github.go

```diff
@@ -245,6 +279,7 @@ func writeResonseToDisk(resp *http.Response, destPath string) *FetchError {
 	defer out.Close()
 	defer resp.Body.Close()
 
-	_, err = io.Copy(out, resp.Body)
+	counter := newWriteCounter(resp.ContentLength)
+	_, err = io.Copy(out, io.TeeReader(resp.Body, counter))
```

Comment:
Should adding this be optional? That is, you can disable (or enable it?) via a CLI flag? 

Main reason might be to reduce log noise when using `fetch` in some script or CI / CD pipeline.

Reply:
It seems like you mean to pass `withProgress` down to this function?

Reply:
fixed